src/main/java/hyperledger/besu/java/rest/client/config/EthConfig.java

```diff
@@ -23,8 +22,7 @@ public class EthConfig {
 
   private List<Web3j> web3jList;
   @Autowired private EthProperties ethProperties;
-
-  @RefreshScope
+  //  @RefreshScope
```

Comment:
Why is the refresh scope commented?

Reply:
This is a workaround to make the application bootstrap, earlier I added a comment, then removed it as it would be handled in Kamini's PR using wrapper.